src/components/common/ga4-opl/template.njk

```diff
@@ -0,0 +1,17 @@
+<script {% if params.nonce%} nonce='{{ params.nonce }}'{%  endif %}>
+    window.addEventListener('load', function () {
+        window
+            .DI
```

Comment:
I think is safer to call `window.DI?.analyticsGa4` just to make sure it doesn't throw if DI is undefined